CotEditor/en-GB.lproj/Acknowledgments.css

```diff
@@ -0,0 +1,45 @@
+
+*     { margin: 0; padding: 0 }
+body  { font: 12px/1.4 -apple-system, sans-serif; padding: 8px 16px }
+h1    { font-size: 18px; font-weight: 600; margin: 0.6em 0 }
+h2    { font-size: 14px; font-weight: 600; margin: 0.2em 0 }
+h3    { font-size: 13px; font-weight: 600; margin: 0.1em 0 }
+em    { font-weight: 600; font-style: normal }
+
+ul    { padding-left: 1.5em }
+ul li { margin: 0 0 0.4em; list-style-type: circle }
+
+a:hover           { text-decoration: none }
+
+section           { margin: 2em 0 1em }
+section+section   { margin-top: 1.5em }
+blockquote pre    { font: 90%/1.3 -apple-system, sans-serif;
+                    margin: 1em 0.5em; white-space: pre-wrap;
+                    border-left: 3px solid; padding-left: 1em }
+h3 span           { font-size: 90%; font-weight: normal }
+
+details           { margin-top: 0.5em }
+details summary   { font-weight: 600; cursor: pointer; outline: none }
+
+h3::before, h2::before   { margin-right: 0.4em }
+#Libraries    h3::before { content: '📦' }
+#InfoSource   h2::before { content: '📚' }
+#Contributors h2::before { content: '👥' }
+#Project      h2::before { content: '🔨' }
+
+a:link,
+a:visited         { color: hsl(90,45%,40%) }
+a:not([href])     { color: hsla( 0, 0%, 0%,0.60) }
+blockquote pre    { color: hsla( 0, 0%, 0%,0.60) ; border-color: hsla( 0, 0%, 0%,0.1) }
+h3 span           { color: hsla( 0, 0%, 0%,0.60) }
+details summary   { color: hsla( 0, 0%, 0%,0.67) }
+
+@media (prefers-color-scheme: dark) {
+    body            { color: hsla(0,0%,100%,0.95); background: hsl( 0, 0%,12%) }
+    a:link,
+    a:visited       { color: hsl(90,55%,55%) }
+    a:not([href])   { color: hsla( 0, 0%,100%,0.60) }
+    blockquote pre  { color: hsla( 0, 0%,100%,0.60) ; border-color: hsla( 0, 0%,100%,0.1) }
+    h3 span         { color: hsla( 0, 0%,100%,0.50) }
+    details summary { color: hsla( 0, 0%,100%,0.67) }
+}
```

Comment:
Because the same name file in the Base.lproj directory is automatically used if not exist in the individual .lporj directory and the file does not contain any language-depended expression, this file can be removed.

Reply:
Fixed in f40005f